src/errors/mod.rs

```diff
@@ -64,15 +64,25 @@ pub fn ceil_char_boundary(value: &str, index: usize) -> usize {
         .map_or(upper_bound, |pos| pos + index)
 }
 
-pub fn write_truncated_to_50_bytes<F: fmt::Write>(f: &mut F, val: Cow<'_, str>) -> std::fmt::Result {
-    if val.len() > 50 {
+pub fn write_truncated_to_limited_bytes<F: fmt::Write>(
+    f: &mut F,
+    val: Cow<'_, str>,
```

Comment:
Should use `&str` here because you don't need ownership.

```suggestion
    val: &str,
```